src/components/_modal/pages/automock_utils.js

```diff
@@ -626,16 +626,20 @@ const USER_DEBT = [
   },
 ];
 
-function useMockLoanerInfo(pickupBranch = "790900") {
+function useMockLoanerInfo(
+  pickupBranch = "790900",
+  loans = USER_LOANS,
+  orders = USER_ORDERS,
+  debt = USER_DEBT
+) {
```

Comment:
Super nice med alle som args, men ville det ikke være nicere at det var et objekt? Så ville det 1-til-1 afspejle formen på det objekt som kommer ind i `updateLoanerInfo`? Det var i hvert fald det jeg mente :sweat_smile:

Reply:
ja, syntes det var lidt cleanere hvsi man ikke har bruge for at give et tomt objekt med når man bruger useMockLoanerInfo, men jeg laver det bare om til et objekt så